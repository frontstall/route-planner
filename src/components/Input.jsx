import React from 'react';
import cn from 'classnames';
import PropTypes from 'prop-types';

const Input = React.forwardRef(({ className, ...props }, ref) => {
  const classes = cn('input', className);

  return <input className={classes} ref={ref} {...props} />;
});

Input.propTypes = {
  className: PropTypes.string,
};

export default Input;
