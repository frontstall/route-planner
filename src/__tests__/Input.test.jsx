import React from 'react';
import renderer from 'react-test-renderer';

import Input from '../components/Input';

it('renders input', () => {
  const component = renderer.create(<Input />).toJSON();
  expect(component).toMatchSnapshot();
});
