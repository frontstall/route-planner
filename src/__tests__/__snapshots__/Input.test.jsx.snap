// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders input 1`] = `
<input
  className="input"
/>
`;
